### ISSUE TITLE
ppv3_rec模型转换不成功

### ISSUE THREAD

**Issue by AstonyJ** [frotms/PaddleOCR2Pytorch#50]
1、转换的pp模型链接：
https://paddleocr.bj.bcebos.com/PP-OCRv3/chinese/ch_PP-OCRv3_rec_train.tar
2、运行脚本：
python ./converter/ch_ppocr_v3_rec_converter.py
3、报错：

**Comment by frotms** [comment]
`python ./converter/ch_ppocr_v3_rec_converter.py --src_model_path paddle_ch_PP-OCRv3_rec_train_dir`

**Comment by AstonyJ** [comment]
> `python ./converter/ch_ppocr_v3_rec_converter.py --src_model_path paddle_ch_PP-OCRv3_rec_train_dir`

我写到py文件里了：
![image](https://user-images.githubusercontent.com/48003410/178929534-3bea1007-8e33-45e5-9059-49d83365f408.png)

**Comment by frotms** [comment]
`python ./converter/multilingual_ppocr_v3_rec_converter.py --src_model_path paddle_multilingual_PP-OCRv3_rec_train_dir`

**Comment by AstonyJ** [comment]
> 

转换成功了 感谢

**Comment by frotms** [comment]
暂时还没整理好，这周末看下

**Comment by AstonyJ** [comment]
> 暂时还没整理好，这周末看下
嗯嗯 感谢呀 我也学习一下 看看能不能转成功

**Comment by frotms** [comment]
@AstonyJ svtr已更新。但请注意paddle的nn.Dropout(p=last_drop, mode="downscale_in_infer")的"downscale_in_infer"参数

**Comment by AstonyJ** [comment]
> 收到 已成功转换 感谢！

**Comment by BLMFY** [comment]
所以楼主这个问题是怎么解决的呢，为什么会出现这个keyerror呢？

**Comment by yangy996** [comment]
> 转换成功了 感谢

楼主这个问题是怎么解决的呢，我现在转换也报这个错
